cluster/ci/config/prow/config.yaml

```diff
@@ -2880,6 +2880,64 @@ presubmits:
         - --artifact-dir=$(ARTIFACTS)
         - --target=unit
         - --target=[images]
+  - name: pull-ci-kubernetes-descheduler-e2e-gce-3.10
+    agent: kubernetes
+    always_run: true
+    context: ci/prow/e2e
+    branches:
+    - release-3.10
+    rerun_command: "/test e2e"
+    trigger: "((?m)^/test( all| e2e),?(\\s+|$))"
+    decorate: true
+    skip_cloning: true
+    spec:
+      serviceAccountName: ci-operator
+      volumes:
+      - name: job-definition
+        configMap:
+          name: prow-job-cluster-launch-src
+      - name: cluster-profile
+        projected:
+          sources:
+          - secret:
+              name: cluster-secrets-gcp
+          - configMap:
+              name: cluster-profile-gcp
+      containers:
+      - name: test
+        image: ci-operator:latest
+        volumeMounts:
+        - name: job-definition
+          mountPath: /usr/local/e2e-gcp
+          subPath: cluster-launch-src.yaml
+        - name: cluster-profile
+          mountPath: /usr/local/kubernetes-descheduler-e2e-cluster-profile
+        env:
+        - name: JOB_NAME_SAFE
+          value: kubernetes-descheduler-e2e
+        - name: CLUSTER_TYPE
+          value: gcp
+        - name: CONFIG_SPEC
+          valueFrom:
+            configMapKeyRef:
+              name: ci-operator-kubernetes-descheduler
+              key: release-3.10.json
+        - name: TEST_COMMAND
+          value: "cp /tmp/admin.kubeconfig /tmp/admin.conf && make test-e2e"
```

Comment:
A bit hack-y, but the test expects this specific path:

https://github.com/openshift/descheduler/blob/d435537acd9224cc8a3912ae8b661f3488a13a33/test/e2e/e2e_test.go#L125

Reply:
We can merge this but can you go upstream to https://github.com/kubernetes-incubator/descheduler and fix this to do something like https://github.com/openshift/ci-vm-operator/blob/master/cmd/ci-vm-operator/ci-vm-operator.go#L102 or at the very least honor the `$KUBECONFIG` var? That e2e code is not the best.

Reply:
@bbguimaraes you OK to take this on?